sdk/appconfiguration/Azure.ApplicationModel.Configuration/tests/ConfigurationLiveTests.cs

```diff
@@ -264,7 +263,6 @@ public async Task GetRequestId()
                 Response<ConfigurationSetting> response = await service.SetAsync(testSetting);
                 response.GetRawResponse().Headers.TryGetValue("x-ms-client-request-id", out string requestId);
                 Assert.IsNotEmpty(requestId);
-                response.Dispose();
```

Comment:
Did you intend to remove this?

Reply:
Yes, talking offline to @KrzysztofCwalina we decided to make Response<T> non-disposable as part of this change.